src/components/esn-popover/esn-popover.tsx

```diff
@@ -22,7 +22,12 @@ export class EsnPopover implements ComponentInterface {
   componentDidLoad() {
     createPopper(this.hostEl, this.popoverContainerEl, {
       strategy: 'fixed',
-      modifiers: [flip, preventOverflow]
+      modifiers: [flip, preventOverflow, {
+        name: 'preventOverflow',
+        options: {
+          padding: 8
```

Comment:
Do we need some 'px' here for the padding option?

Reply:
You can't pass `padding: 8px` here. The expected type here is `number`, and the unit will be `px`.